src/dom-builder.ts

```diff
@@ -74,10 +82,7 @@ export type InitialProperties = {
   actionDefinitions?: ActionDefinitions,
   actions?: Actions,
   setters?: { [name: string]: Behavior<any> },
-  class?: string,
-  classToggle?: {
-    [name: string]: boolean | Behavior<boolean>;
-  }
+  class?: ClassDescription | ClassDescription[]
```

Comment:
The code handles arrays recursively but the types do not.

Maybe `ClassDescription` should be like this?

```ts
type ClassDescription = ClassNames | ClassToggles | ClassDescription[];
```

Reply:
Since Typescript tries to unfold a type declaration, I had to implement it using an interface for the arrays

```ts
type ClassDescription = ClassNames | ClassToggles | ClassDescriptionArray;
interface ClassDescriptionArray extends Array<ClassDescription> {}
```